lib/pbio/src/protocol/pybricks.c

```diff
@@ -87,9 +87,17 @@ const uint16_t pbio_gatt_device_info_service_uuid = 0x180A;
 
 /** Bluetooth Firmware Version Characteristic UUID. */
 const uint16_t pbio_gatt_firmware_version_char_uuid = 0x2A26;
+const uint8_t pbio_gatt_firmware_version_char_uuid_128[] = {
```

Comment:
Or, even better, don't add the 128-bit version for 16-bit UUIDs and make a new copy-uuid16-to-rfc4122 function instead.

Reply:
The Bluetooth driver already includes `uuid_add_bluetooth_prefix`, so I just used that and the `pbio_uuid128_le_copy` already added. I have to manually copy the function declaration because including the header (btstack_util.h) wasn't clean. I can combine these into a single "copy-ble-uuid16-to-usb-uuid128" function instead if you think that would be better.